Phonebook.Frontend/src/app/pages/page-information/page-information.component.scss

```diff
@@ -1,14 +1,29 @@
-mat-card-content {
-  margin: 0 16px;
+@import 'mixins';
+@import 'media-queries';
+@import '~@angular/material/theming';
 
-  h2 {
-    margin: 16px 0 0 0;
-  }
+//center all text
+* {
+  text-align: center;
+}
 
-  #contribute {
-    display: flex;
-    span {
-      margin-left: 10px;
+mat-card {
+  margin: 0.5%;
+  border: 1.5px solid;
+
+  mat-card-header {
+    //place for the icon
```

Comment:
"Platz für das Icon"?

Reply:
I mean that `mat-card-header` is the place where the icon is inside.
Don't know how to comment this

Reply:
Ich meinte damit wenn du nur das hinschreibst was offensichtlich ist kannst du ein Kommentar auch weglassen.